src/seedu/addressbook/Main.java

```diff
@@ -88,9 +90,23 @@ private void runCommandLoopUntilExitCommand() {
             recordResult(result);
             ui.showResultToUser(result);
 
+            enhancedAddnDelete(command);
+
+
+
         } while (!ExitCommand.isExit(command));
     }
 
+    /** List all persons in contact when a Add/ Delete Command is entered */
+    private void enhancedAddnDelete(Command command){
+        if(command instanceof AddCommand | command instanceof DeleteCommand){
+            command = new Parser().parseCommand("list");
+            CommandResult nextResult = executeCommand(command);
+            recordResult(nextResult);
+            ui.showResultToUser(nextResult);
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.